eng/pipelines/extra-platforms/runtime-extra-platforms-ioslike.yml

```diff
@@ -90,11 +90,9 @@ jobs:
         - template: /eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml
           parameters:
             creator: dotnet-bot
-            compileOnHelix: true
-            interpreter: true
-            testBuildArgs: /p:ArchiveTests=true /p:DevTeamProvisioning=- /p:RunAOTCompilation=true /p:MonoForceInterpreter=true /p:BuildTestsOnHelix=true
+            testBuildArgs: /p:DevTeamProvisioning=- /p:RunAOTCompilation=true /p:MonoForceInterpreter=true
             testRunNamePrefixSuffix: Mono_$(_BuildConfig)
-            extraHelixArguments: /p:NeedsToBuildAppsOnHelix=true
```

Comment:
@jkoritzinsky Did you intentionally remove `NeedsToBuildAppsOnHelix`?

Reply:
I was trying it as an option to see if it would work and be easier to debug. If you prefer, we can switch back to that.